cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export const ActionConfig = [
+  {
+    img: '/cdap_assets/img/cleanse_data.svg',
+    title: 'Cleanse data',
+    description:
+      'Preview data from external sources. Prototype data transformations in realtime in Wrangler.',
+    links: [
+      {
+        label: 'Wrangler',
+        url: `/cdap/ns/:namespace/wrangler`,
+      },
+    ],
+  },
+  {
+    img: '/cdap_assets/img/data_pipelines.svg',
+    title: 'Build and manage data pipelines',
+    description: 'Build pipelines in Studio. Manage deployed and draft pipelines in List.',
+    links: [
+      {
+        label: 'Studio',
+        url: '/pipelines/ns/:namespace/studio',
+      },
+      {
+        label: 'List',
+        url: '/cdap/ns/:namespace/pipelines',
+      },
+    ],
+  },
+  {
+    img: '/cdap_assets/img/metadata.svg',
+    title: 'Explore metadata',
+    description: 'Search by dataset. View schema, lineage, properties, and tags.',
+    links: [
+      {
+        label: 'Metadata',
+        url: '/metadata/ns/:namespace',
+      },
+    ],
+  },
+  {
+    img: '/cdap_assets/img/operations.svg',
+    title: 'Monitor operations',
+    description: 'Monitor pipeline runs in Dashboard.',
+    links: [
+      {
+        label: 'Dashboard',
+        url: '/cdap/ns/:namespace/operations',
+      },
+    ],
+  },
+  {
+    img: '/cdap_assets/img/administration.svg',
+    title: 'Administrations',
```

Comment:
Should probably be "Administration" without the s to match rest of UI?

Reply:
I'm not sure about this. @leacd any input?